AEPIdentity/Sources/PushIDManager.swift

```diff
@@ -48,7 +48,7 @@ struct PushIDManager: PushIDManageable {
     }
 
     mutating func updatePushId(pushId: String?) {
```

Comment:
I think we can use `pushId: String` here as we are doing the check in IdentityState syncIdentifiers

Reply:
We need to be able to pass `nil` in the case we are removing the push id in case of setting to opt out.